src/Core/Enums/PersonaImagePosition.cs

```diff
@@ -0,0 +1,17 @@
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// Describes the position at which a <see cref="FluentPersona"/> image is shown.
+/// </summary>
+public enum PersonaImagePosition
+{
+    /// <summary>
+    /// The tooltip is positioned to the left of the anchor element.
+    /// </summary>
+    Left,
```

Comment:
As in my previous comment, and to follow Vincent's logic, it's probably better to use the names `Before` and `After`.

Reply:
We use start/end throughout the library (icon slots, alignment, ...) so I would stick with that. Don't recall we use before/after anywhere